lib/ga-track.js

```diff
@@ -84,7 +84,9 @@
 
       el.on('click', function(e) {
         $.gaTrack(cat, action, label);
-        if (href && !e.metaKey && e.which === 1) {
+        if (el.data('ga-track-href') === false) {
```

Comment:
jquery evaluates, which I just found out. Can even put json in there and it'll become an object.